web/src/components/ImageInfoDialog.js

```diff
@@ -1,35 +1,46 @@
 import React from 'react'
 import Dialog from '@material-ui/core/Dialog'
-import { DialogContent, DialogActions, Button, DialogTitle, Typography } from '@material-ui/core';
+import { DialogContent, DialogActions, List, Button, DialogTitle, Typography, ListItem, ListItemText } from '@material-ui/core';
 
 class ImageInfoDialog extends React.Component {
-    getRegistry() {
-        let arr = this.props.docker.image.split('/')
+    getRegistry(docker) {
+        let arr = docker.image.split('/')
         return arr[0]
     }
 
-    getRepo() {
-        let arr = this.props.docker.image.split('/')
+    getRepo(docker) {
+        let arr = docker.image.split('/')
         return arr[1]
     }
 
-    getTag() {
-        return this.props.docker.tag
+    getTag(docker) {
+        return docker.tag
     }
 
-    getURI() {
-        return this.props.docker.image + ':' + this.getTag()
+    getURI(docker) {
+        return docker.image + ':' + this.getTag()
     }
 
     render() {
+        this.images = this.props.docker.map(d => 
+            <div>
+                <ListItem>
+                    <ListItemText>
+                        <Typography><b>Registry:</b> {this.getRegistry(d)}</Typography>
```

Comment:
I tend to use <label> and style it rather than a <b> tag for this kind of thing, but this is ok for now.

Reply:
I will push a fix to use a styled `<label>` tag to get the same effect as `<b>`.